doc/conf.py

```diff
@@ -50,7 +50,7 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'spikeinterface'
+project = 'Spikeinterface'
```

Comment:
I think I prefer lower case.

Reply:
I did it so it will be capitalized in the docs in the corner:
![image](https://user-images.githubusercontent.com/6429509/234266457-e77b91e8-14b0-4a51-8223-16172b5ebea2.png)

So, it is consistent with the main title.